src/client/editor/nodes/SpawnPointNode.js

```diff
@@ -11,8 +11,7 @@ export default class SpawnPointNode extends EditorNodeMixin(THREE.Object3D) {
 
   static async load() {
     const { scene } = await new Promise((resolve, reject) => {
-      const loader = new THREE.GLTFLoader();
-      loader.load(spawnPointModelUrl, resolve, null, reject);
+      new THREE.GLTFLoader().load(spawnPointModelUrl, resolve, null, reject);
```

Comment:
Probably don't want to create a new loader every time. Also this doesnt use `this` so it should just bbe a function. Might also make sense to just load once it at startup and then clone, since right now you end up re-parsing the gltf for each spawn point.

Reply:
This one is actually only loaded once at startup and then cloned. I'm going to keep it as is.